src/modules/functionality.js

```diff
@@ -0,0 +1,69 @@
+// ADD FUNCTIONALITY
+
+const add = (tasks) => {
+  const container = document.querySelector('.container');
+  tasks.forEach((task) => {
+    const taskEl = `
+      <li class="list-item"><div class="left"><input type="checkbox" id=${task.index} name=${task.description} value=${task.description}>
+      <label for=${task.index} hidden>${task.description}</label><textarea class="edit" maxlength="255" spellcheck="false" focus>${task.description}</textarea></div><div class="right"><i class="fa-solid fa-ellipsis-vertical"></i><i class="fa-solid fa-trash-can delete"></i></div>
+      </li>`;
+
+    container.innerHTML += taskEl;
+  });
+};
+
+// REMOVE FUNCTIONALITY
+
+const removeList = (taskArray, fillTask, el) => {
+  const index = parseInt(el.children[0].children[0].getAttribute('id'), 10);
+  taskArray.splice(index, 1);
+  taskArray.forEach((entry, i) => {
+    entry.index = i;
+  });
+  fillTask();
+  el.remove();
+};
```

Comment:
- [ ] Please make sure to implement this functionality properly. Your function is defined, but you are not exporting it so it's not being used by your index.js file to delete tasks.

Reply:
Hi @Lordkaito, Thanks for the review, will implement that change right away 📝